bfvmm/include/vmcs/vmcs_intel_x64.h

```diff
@@ -294,6 +299,8 @@ class vmcs_intel_x64
 
     friend class vmcs_ut;
 
+    bitmap *m_msr_bitmap;
```

Comment:
Why is the bitmap a pointer? At minimum, it should be a make_shared, but I would imagine this could be done without a pionter